src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/PreferencesDialog.css

```diff
@@ -27,16 +27,28 @@
    margin-bottom: 11px;
 }
 
+.paneLayoutTable {
+   height: 160px;
+}
+
 .paneLayoutTable td.paneLayoutTable {
    background-color: white;
-   height: 200px;
    vertical-align: top;
-   padding-top: 0px;
+   padding: 0px;
+   border: 1px solid #1e77bc;
```

Comment:
Maybe personal preference but I think it looks a little odd to have square corners (from the options dropdowns) right up against these rounded ones. Can we tone these borders down a little (just a dark gray) and reduce the radius?

Reply:
I changed the border and added a screenshot to this PR's description.